homeassistant/components/ta_cmi/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "ta_cmi",
+  "name": "Technische Alternative C.M.I.",
+  "config_flow": false,
```

Comment:
New integrations that communicate with devices and/or services should be configurable with a Config Flow instead of YAML.

For more detail read [ADR-0010](https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision)

Reply:
The documentation is available on https://developers.home-assistant.io/docs/config_entries_config_flow_handler.
The scaffold script can help you set up everything you need:
```
python3 -m script.scaffold integration
```

Reply:
I added it in the new commit